web_src/js/features/clipboard.js

```diff
@@ -1,21 +1,18 @@
 // For all DOM elements with [data-clipboard-target] or [data-clipboard-text], this copy-to-clipboard will work for them
 
-// TODO: replace these with toast-style notifications
 function onSuccess(btn) {
-  if (!btn.dataset.content) return;
   $(btn).popup('destroy');
   const oldContent = btn.dataset.content;
-  btn.dataset.content = btn.dataset.success;
+  btn.dataset.content = btn.dataset.success || '';
   $(btn).popup('show');
-  btn.dataset.content = oldContent;
+  btn.dataset.content = oldContent || '';
```

Comment:
The change seems not necessary. Can the `btn.dataset.content` be set to `oldContent` directly?

Reply:
It's neccessary because otherwise the popups show `undefined` for buttons without `data-content` which now pass this code with the `if` guard removed above.